Pipfile

```
@@ -17,3 +17,10 @@ pytest = "*"
 
 [requires]
 python_version = "3.9"
+
+[scripts]
+manage_pip = "python manage.py"
+shell_pip = "python manage.py shell"
+run_pip = "python manage.py runserver 0.0.0.0:8000"
+makemigrations_pip = "python manage.py makemigrations"
+migrate_pip = "python manage.py migrate"
```

if you can do it here like:
```suggestion
migrate = "pipenv run python manage.py migrate"
```
it could be great!!

or if you can do it in the bashrc like:
alias migrate="pipenv run python manage.py migrate"
IMHO no need to mix it.
when you mix it doing part of the command here and the second part in bashrc it make it more complicate